### ISSUE TITLE
GitHub integration pulling wrong repo

### ISSUE THREAD

**Issue by Morganamilo** [johannesjo/super-productivity#55]
## Your Environment

* Version used: 1.7.5.r0.g20d6d67-1 
* Operating System and version: 4.15.3-2-ARCH
* Desktop Environment: i3

## Expected Behaviour

If the git integration is invalid to nothing.

## Current Behavior

When using git integration, if you enter a non valid git repo for `Git repository to track for importing tasks` it seems to fall back on the git repo used in another project (last switched, last created I'm unsure) If auto import is enabled it will also import all the issues from that repo.

This could also apply to more things such as `Set project directory to show the day's commits at summary.` but I have not tested those.

## Steps to Reproduce (for bugs)

1. Create project with valid git integration
2. Create a new project with an invalid git integration (box can't be empty) and auto import
3. watch as tasks are created using issues from the repo in project 1

**Comment by johannesjo** [comment]
Hey there! Thanks for reporting! By invalid do you mean empty?

**Comment by Morganamilo** [comment]
No, I did mention it in the "Steps to Reproduce" but forgot to mention it elsewhere. 

A while had one project set to "johannesjo/super-productivity" just to test the feature. Then recently I wanted to use it for real so I made a project and entered "https://github.com/Jguer/yay". I guess you don't support entering the full URL because it started pulling in issues from this repo.

I tested again with just "a" in the box and it did the same thing.

When the box is empty it does nothing at all.

Looking at that commit you already fixed it though, maybe the extra info will be useful anyway.
